kubric/core/objects.py

```diff
@@ -284,4 +284,11 @@ class FileBasedObject(PhysicalObject):
   # UI, minus a 90 degree X-axis rotation applied after loading.
   glb_do_transform_apply_after_import = tl.Bool(False)
 
+  # If true, uses a parenting approach instead of a join so that the asset
+  # represents a top-level Empty node in the scene. All existing objects
+  # without parents will be parented to this object. Any transform applied to
+  # this object will be applied to all children. For more details, see
```

Comment:
Two comments:
  * This will likely break assigning materials to the object. Not a big problem, because material was  never well supported, but probably worth adding a comment about here. 
  * The parenting flag only applies to usage of the object in blender. That means that pybullet will still use the regular collision mesh which is definitely not animated. So some care should be taken when using animated assets together with a physics simulation. (Not really a problem of the flag, or even the parenting based method, but might still be nice to mention here).

Reply:
Thank you for the suggestions. Extra care should also be taken when assigning things like `segmentation_ids`. I've made a note about this.